test/new_order.spec.ts

```diff
@@ -192,11 +209,15 @@ describe('NewOrder', function () {
       expect(order.makerAddress).eq('0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852')
       expect(order.makerAssetAmount).eq('100000')
       expect(order.makerAssetAddress).eq('0xdac17f958d2ee523a2206206994597c13d831ec7')
-      expect(order.makerAssetData).eq('0xf47261b0000000000000000000000000dac17f958d2ee523a2206206994597c13d831ec7')
+      expect(order.makerAssetData).eq(
+        '0xf47261b0000000000000000000000000dac17f958d2ee523a2206206994597c13d831ec7'
```

Comment:
lots of blank line in this file, would be better to format this file

Reply:
Resolved